KhronosGroup/Vulkan-LoaderAndValidationLayers
# Update to version 1.1.73 of the Vulkan header

Note that the registry barely changed for this update, but a big handful of VUIDs were removed, which had a ripple effect.

- removed checks for the following deprecated VUIDs related to ImageCreateInfo and FormatProperties:
     - VALIDATION_ERROR_09e007ae
     - VALIDATION_ERROR_09e007b4
     - VALIDATION_ERROR_09e007ac
     - VALIDATION_ERROR_09e007a6
     - VALIDATION_ERROR_09e007b2
     - VALIDATION_ERROR_09e007a4
     - VALIDATION_ERROR_09e007aa
     - VALIDATION_ERROR_09e007a8
     - VALIDATION_ERROR_09e007a2
     - VALIDATION_ERROR_09e007b0
- modified CreateImageFormatSupportErrors test to remove killed checks
- modified ImageLayerUnsupportedFormat test to remove killed check
- update json files
- update vk.xml
- update vulkan_core.h
- update validusage.json
- update vk_validation_error_messages.h
- update vk_validation_error_database.txt
- update vulkan.hpp
